hic/testdata/strawTestFile.straw

```diff
@@ -0,0 +1,2 @@
+1000	5000	5
```

Comment:
Might be good to switch Bin1Start and Bin2Start for one of these (to test for the lack of restriction on ordering)

Reply:
I changed the values in the data struct instead of the file